src/scheduling.py

```diff
@@ -40,16 +40,17 @@ def logout():
         response.set_cookie(USER_ID_COOKIE, "", expires=0)
         return response
 
-    @app.route("/login", methods=["POST"])
+    @app.route("/login", methods=["POST"])  # using POST method for user login
```

Comment:
Method documentation is typically in doc strings. Can you move these into triple quote comments directly inside the function?
def login():
   """using POST method for user login"""

Reply:
Yes I will change it to triple quote comments.
Actually this is my first time contributing to open source.
So I hope you understand it.

Reply:
No worries, we're happy to have you contribute.

Reply:
Thank you.

Reply:
I wanted to know that should I comment in triple quotes only?